builtin_global.go

```diff
@@ -327,28 +329,85 @@ func (r *Runtime) builtin_unescape(call FunctionCall) Value {
+func createGlobalObjectTemplate() *objectTemplate {
+	t := newObjectTemplate()
+	t.protoFactory = func(r *Runtime) *Object {
+		return r.global.ObjectPrototype
+	}
+
+	t.putStr("Object", func(r *Runtime) Value { return valueProp(r.getObject(), true, false, true) })
+	t.putStr("Function", func(r *Runtime) Value { return valueProp(r.getFunction(), true, false, true) })
+	t.putStr("Array", func(r *Runtime) Value { return valueProp(r.getArray(), true, false, true) })
+	t.putStr("String", func(r *Runtime) Value { return valueProp(r.getString(), true, false, true) })
+	t.putStr("Number", func(r *Runtime) Value { return valueProp(r.getNumber(), true, false, true) })
+	t.putStr("RegExp", func(r *Runtime) Value { return valueProp(r.getRegExp(), true, false, true) })
+	t.putStr("Date", func(r *Runtime) Value { return valueProp(r.getDate(), true, false, true) })
+	t.putStr("Boolean", func(r *Runtime) Value { return valueProp(r.getBoolean(), true, false, true) })
+	t.putStr("Proxy", func(r *Runtime) Value { return valueProp(r.getProxy(), true, false, true) })
+	t.putStr("Reflect", func(r *Runtime) Value { return valueProp(r.getReflect(), true, false, true) })
+	t.putStr("Error", func(r *Runtime) Value { return valueProp(r.getError(), true, false, true) })
+	t.putStr("AggregateError", func(r *Runtime) Value { return valueProp(r.getAggregateError(), true, false, true) })
+	t.putStr("TypeError", func(r *Runtime) Value { return valueProp(r.getTypeError(), true, false, true) })
+	t.putStr("ReferenceError", func(r *Runtime) Value { return valueProp(r.getReferenceError(), true, false, true) })
+	t.putStr("SyntaxError", func(r *Runtime) Value { return valueProp(r.getSyntaxError(), true, false, true) })
+	t.putStr("RangeError", func(r *Runtime) Value { return valueProp(r.getRangeError(), true, false, true) })
+	t.putStr("EvalError", func(r *Runtime) Value { return valueProp(r.getEvalError(), true, false, true) })
+	t.putStr("URIError", func(r *Runtime) Value { return valueProp(r.getURIError(), true, false, true) })
+	t.putStr("GoError", func(r *Runtime) Value { return valueProp(r.getGoError(), true, false, true) })
+
+	t.putStr("eval", func(r *Runtime) Value { return valueProp(r.getEval(), true, false, true) })
+
+	t.putStr("Math", func(r *Runtime) Value { return valueProp(r.getMath(), true, false, true) })
+	t.putStr("JSON", func(r *Runtime) Value { return valueProp(r.getJSON(), true, false, true) })
+	addTypedArrays(t)
+	t.putStr("Symbol", func(r *Runtime) Value { return valueProp(r.getSymbol(), true, false, true) })
+	t.putStr("WeakSet", func(r *Runtime) Value { return valueProp(r.getWeakSet(), true, false, true) })
+	t.putStr("WeakMap", func(r *Runtime) Value { return valueProp(r.getWeakMap(), true, false, true) })
+	t.putStr("Map", func(r *Runtime) Value { return valueProp(r.getMap(), true, false, true) })
+	t.putStr("Set", func(r *Runtime) Value { return valueProp(r.getSet(), true, false, true) })
+	// todo Promise isn't needed
```

Comment:
[nit] We usually mark changes on our branch with `// DIVERSION:`

```suggestion
	// DIVERSION: we are not supporting Goja Promises
```